src/plugins/controls/common/options_list/types.ts

```diff
@@ -20,46 +20,52 @@ export interface OptionsListEmbeddableInput extends DataControlInput {
   existsSelected?: boolean;
   runPastTimeout?: boolean;
   singleSelect?: boolean;
+  hideActionBar?: boolean;
   hideExclude?: boolean;
   hideExists?: boolean;
+  hideFooter?: boolean;
```

Comment:
cc @logeekal  Because we were adding the cardinality badge beside the search bar, the action bar was starting to get pretty messy (especially when you also had ignored selections). So, we decided to move the "Clear all selections" and "Show only selected" buttons to the **footer**, like so:

![Screenshot 2023-01-30 at 11 34 47 AM](https://user-images.githubusercontent.com/8698078/215565194-5ae28861-8a33-40cb-8617-783c2fe5a17c.png)

Previously, the `hideExcludes` property would hide the entire footer - however, now that some other buttons have been moved to this footer, that no longer makes sense. Since I know that consumers of the `ControlGroupRenderer` want control over which parts of the popover are shown, I went ahead and added the `hideFooter` property, which should be used **instead** of the `hideExcludes` property if you want the entire footer to be hidden:

![image](https://user-images.githubusercontent.com/8698078/215564633-8da53fbe-6315-4b3a-8140-fcad0a9859bd.png)

The `hideExcludes` property now literally just hides the include/exclude button group, like so:

![image](https://user-images.githubusercontent.com/8698078/215564737-7ce1b713-59bb-4192-b64b-7cf8482c3d9e.png)

Note that there is **no UI** to set these properties - these are intended specifically for consumers.

Reply:
cc @logeekal

Reply:
**Update:** We have moved to the following design to display the cardinality rather than a badge:

![image](https://user-images.githubusercontent.com/8698078/216449179-ab913463-2608-49b4-b50c-1a459c48b09b.png)

So, I've modified the code to make `hideExclude` once again hide the entire footer, which makes `hideFooter` no longer necessary. Resolved in 6b7ab49dac294ddd3066674d5c429433a064bdf7